python/create_service_diagnostics_bundle.sh

```diff
@@ -67,10 +66,10 @@ mkdir -p "${BUNDLES_DIRECTORY}"
 function container_run () {
   local -r command="${*:-}"
   docker run \
-         -it \
+         -t \
```

Comment:
Is the interactive flag needed when the script isn't run with the `--yes` flag?

Reply:
I'd prefer to keep this outside the scope of this PR for easier rollbackability, but... meh.

Reply:
Do you think it will break the script on the scenario above though? I haven't tested it myself yet.

Reply:
I was tripped up so many times when thinking "this will *obviously* work, so no need to plan for failure" that I prefer to tread carefully.